src/renderer/entities/wallet/model/wallet-model.ts

```diff
@@ -233,7 +295,11 @@ export const walletModel = {
     multisigCreated,
     walletConnectCreated,
     walletRemoved,
+    walletHidden,
+    walletHiddenSuccess: hideWalletFx.done,
```

Comment:
I'd rather declared `output: {}` where we put output events that are readonly
We already do so in `network-model`

Reply:
In governance, we’ve started placing these requests in events, as their usage is no different from any other event